contrib/chinese/packages.el

```diff
@@ -14,13 +14,17 @@
 ;; which require an initialization must be listed explicitly in the list.
 (setq chinese-packages
     '(
-      youdao-dictionary
-      chinese-pyim
       find-by-pinyin-dired
       ace-pinyin
-      chinese-wbim
       ))
 
+(if chinese-enable-youdao-dict
+  (add-to-list 'chinese-packages 'youdao-dictionary))
```

Comment:
Thanks ,@person808  Any reference to this topic? I found some any usage in spacemacs.
That's why I choose `add-to`list`.